### ISSUE TITLE
npm install error

### ISSUE THREAD

**Issue by 87182485** [transitive-bullshit/ffmpeg-concat#87]
```
npm ERR! path /home/garyyang/.nvm/versions/node/v10.13.0/lib/node_modules/ffmpeg-concat/lib/cli
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/garyyang/.nvm/versions/node/v10.13.0/lib/node_modules/ffmpeg-concat/lib/cli'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/garyyang/.npm/_logs/2020-07-23T04_20_14_916Z-debug.log
```

OS: Linux Mint, 
node version: currently using nvm, tried both 10.13.0 and 12.18.2

**Comment by 87182485** [comment]
Quick update on this one, seems like version 1.1.0 working fine for me.

**Comment by transitive-bullshit** [comment]
Fixed in `v1.2.3`.

**Comment by nhuky1988** [comment]
> PS C:\Users\ZING\Downloads\ffmpeg-concat-master> node .\index.js                                                        ffmpeg-concat: 88.844ms
(node:12872) UnhandledPromiseRejectionWarning: Error: spawn ffprobe ENOENT
    at notFoundError (C:\Users\ZING\Downloads\ffmpeg-concat-master\node_modules\cross-spawn\lib\enoent.js:6:26)
    at verifyENOENT (C:\Users\ZING\Downloads\ffmpeg-concat-master\node_modules\cross-spawn\lib\enoent.js:40:16)
    at ChildProcess.cp.emit (C:\Users\ZING\Downloads\ffmpeg-concat-master\node_modules\cross-spawn\lib\enoent.js:27:25)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
(node:12872) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:12872) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

I RUN AT THIS WINDOWS 10? HELP ME TO THIS ERROR
